packages/dd-trace/test/exporters/common/request.spec.js

```diff
@@ -216,8 +218,8 @@ describe('request', function () {
     })
   })
 
-  it('should be able to send concurrent requests to different hosts', function (done) {
-    this.timeout(10000)
+  // TODO: make this work regardless of the test runner
+  it.skip('should be able to send concurrent requests to different hosts', function (done) {
```

Comment:
why doesn't it work with tap? This was protecting us against a nasty regression

Reply:
It seems to be hanging the test for some reason when enabled. There are things that mocha does, like killing the process when tests are done, that tap doesn't do so when something is not cleaning up it can cause the process to never end.

Reply:
I think all that was needed was to make sure the servers are closed, just pushed a fix.